### PR TITLE
docs: first theorem

### LINKED ISSUE #22: Comparing greater than against rounded down number
https://github.com/morpho-labs/morpho-utils/blob/edc94386e191c44ec4dc2b596ee5185758febc63/src/math/WadRayMath.sol#L31

The noted comparison compares in solidity against a rounded down number. I don't see either aave or solmate using this kind of comparison, so I'm not sure it's kosher. Might be leaky?

### PR DESCRIPTION
**Pull request by MathisGD**
Adds a pdf document for useful theorems and proofs for the repo. 
Fixes #22.

### CODE REVIEW

**Review by MerlinEgalite — APPROVED** [review]
LGTM

File: docs/morpho-utils.tex [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+\begin{proof}
+    Let $M\in\mathbb{N}, (x, y)\in{\mathbb{N}^*}^2$, the case where $x$ or $y$ equals to 0 being trivial. 
+    With lemma \ref{lemma-1}, we have:
+    \begin{align*}
+        x > \lfloor M/y \rfloor &\Leftrightarrow x > M/y \\
+    \end{align*}
+    So:
+    \begin{align*}
+        x > \lfloor M/y \rfloor &\Leftrightarrow x \times y > M\\
+        &\Leftrightarrow x \times y > M\\
```
Comment (QGarchery) [inline docs/morpho-utils.tex:55]: The equivalence $$\Leftrightarrow x \times y > M$$ is duplicated

File: docs/morpho-utils.tex [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    Now, we suppose that $\lfloor x\rfloor<n$. We have:
+    \[\lfloor x\rfloor\leq x<\lfloor x\rfloor+1\]
+    Also, because $\lfloor x\rfloor$ is an integer, we have:
+    \[\lfloor x\rfloor+1\leq n\]
+    So:
+    \[x<n\]
+\end{proof}    
+
+\begin{theorem}
+    \[\forall(x,y, M)\in\mathbb{N}^3, x>\lfloor M/y \rfloor \Leftrightarrow x\times y>M\]
```
Comment (QGarchery) [inline docs/morpho-utils.tex:43]: `y` should not be 0, or the theorem has no meaning in this case

**Review by QGarchery — COMMENTED** [review]
Awesome, let's do more of these :)

**Review by QGarchery — APPROVED** [review]

**Pull request merged**